gateway/gateway.yaml

```diff
@@ -106,6 +106,8 @@ georchestra:
       header: 
         target: http://header:8080/header/
         access-rules:
+        - intercept-url: /header/ogcproxy/**
```

Comment:
there's no ogcproxy in /header to my knowledge

Reply:
ça existe uniquement dans security-proxy (bean spMetadataSource de type securityMetadataSource).
Dans le header path ogcproxy/ n'existe plus aprés un code check rapide!

Reply:
This change seems to have no effect.

Reply:
Please remove